src/main/kotlin/racingcar/view/InputView.kt

```diff
@@ -0,0 +1,18 @@
+package racingcar.view
+
+class InputView private constructor() {
+
+    companion object {
+        fun inputCarCount(): Int {
+            println("자동차 대수는 몇 대인가요?")
+
+            return readLine()!!.toInt()
```

Comment:
```suggestion
            return readln().toInt()
```
코틀린 1.6부터 추가된 새로운 함수입니다!
+) 입력값이 Int가 아닐 경우에 커스텀 예외를 제공하고 싶다면 `toIntOrNull`을 활용할 수 있을 것 같아요. 이처럼 코틀린은 제공하는 함수들에 null/non-null을 리턴하는 함수 두 개를 모두 제공하는 경우가 많습니다. 상황에 맞게 사용해주시면 될 것 같아요.

참고로, non-null assertion 연산자를 !!처럼 불편한 모양으로 정한 건, 사용할 때 한 번 더 생각해보고 불편하게 사용하라는 코틀린 개발진들의 의도가 들어가있다고 합니다 😄 코틀린이 제공하는 기본 함수에 이를 사용하려고 한다면 non-nullable한 리턴값을 갖는 동일한 함수가 있는지 확인해봐도 좋을 것 같아요.

Reply:
저만 느끼기에 !!이 불편했던게 아니군요, 좋은 팁 감사합니다.🙏